docs-src/concepts/what-is-a-timer.md

```diff
@@ -0,0 +1,25 @@
+---
+id: what-is-a-timer
+title: What is a Timer?
+sidebar_label: Timer
+description: A Workflow can set a durable timer for a fixed time.
```

Comment:
I think this could either be slightly more accurate (see my comment below) or more descriptive of the expectations someone might have with the content such as "Learn more about Temporal's Durable Timers"